### PR TITLE
Desktop portfolio version

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
In this project a desktop portfolio version was created . the page is responsive both on mobile and desktops screens. no complex code is done here.
Built With

- HTML
- CSS
- Linters

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -532,3 +531,206 @@ input[type="submit"]:disabled {
...
+    text-align: center;
+    position: relative;
+    max-width: 600px;
+    margin-left: 10px;
+  }
+
+  /* to position hr */
+  .first-box > .ohh {
+    display: block;
+    position: relative;
```
Comment (sja-thedude) [inline style.css]: - [ ] Kindly make sure to follow the Figma design and make all the changes according to the loom video attached.
Like the spacing, font family, font size, hover on the cards etc.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Ngufor-emmanuella, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct Gitflow
✔️ All linters passing

## Required 
- [ ] Kindly make sure to make all the changes from [here](https://www.loom.com/share/76c98f42d74846389b37e6769731224b).
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Ngufor-emmanuella) [reply style.css]: noted thanks

File: style.css [hunk 2]
```diff
@@ -532,3 +533,255 @@ input[type="submit"]:disabled {
...
+    margin-left: 80px;
+  }
+
+  #headline > p {
+    font-size: 16px;
+    width: 55%;
+    margin-top: 5px;
+    margin-left: 80px;
+    line-height: 2rem;
+  }
```
Comment (franclobo) [inline style.css]: - [ ] **[OPTIONAL]** Please 🙏🏽 pay attention to the suggerences from the last code reviewer:

> Kindly make sure to make all the changes from [here](https://www.loom.com/share/76c98f42d74846389b37e6769731224b).

- [ ] **[OPTIONAL]** Kindly correct the elements in red to be similar to the Figma template 👇🏽 

image

File: style.css [hunk 3]
```diff
@@ -532,3 +533,255 @@ input[type="submit"]:disabled {
...
+  #work > .first-box > .headtext > h2 {
+    border-bottom: none;
+    font-family: 'Inter', sans-serif;
+    font-size: 30px;
+    font-weight: 500;
+    text-align: center;
+    position: relative;
+    max-width: 600px;
+    margin-left: 10px;
+  }
```
Comment (franclobo) [inline style.css]: - [ ] **[OPTIONAL]** Please 🙏🏽 pay attention to the suggerences from the last code reviewer:

> Kindly make sure to make all the changes from [here](https://www.loom.com/share/76c98f42d74846389b37e6769731224b).

- [ ] **[OPTIONAL]** Kindly correct the elements in red to be similar to the Figma template 👇🏽 
image
image

File: style.css [hunk 4]
```diff
@@ -532,3 +533,255 @@ input[type="submit"]:disabled {
...
+
+  .ttwork {
+    height: 19%;
+    width: 40%;
+  }
+
+  #work > .carttt {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
```
Comment (franclobo) [inline style.css:815]: - [ ] **[OPTIONAL]** Please 🙏🏽 pay attention to the suggerences from the last code reviewer:

> Kindly make sure to make all the changes from [here](https://www.loom.com/share/76c98f42d74846389b37e6769731224b).

- [ ] **[OPTIONAL]** Kindly correct the elements in red to be similar to the Figma template 👇🏽 
image

File: style.css [hunk 5]
```diff
@@ -532,3 +533,255 @@ input[type="submit"]:disabled {
...
+
+  #about {
+    background-image: url("Resources/Images/illustration-aboutme.png");
+    background-size: 50%;
+    background-position-y: 85px;
+  }
+
+  #about > p {
+    width: 50%;
+  }
```
Comment (franclobo) [inline style.css]: - [ ] **[OPTIONAL]** Please 🙏🏽 pay attention to the suggerences from the last code reviewer:

> Kindly make sure to make all the changes from [here](https://www.loom.com/share/76c98f42d74846389b37e6769731224b).

- [ ] **[OPTIONAL]** Kindly correct the elements in red to be similar to the Figma template 👇🏽 
image

File: style.css [hunk 6]
```diff
@@ -532,3 +533,255 @@ input[type="submit"]:disabled {
...
+
+  #about > .line {
+    width: 100%;
+  }
+
+  .listing {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+  }
```
Comment (franclobo) [inline style.css]: - [ ] **[OPTIONAL]** Please 🙏🏽 pay attention to the suggerences from the last code reviewer:

> Kindly make sure to make all the changes from [here](https://www.loom.com/share/76c98f42d74846389b37e6769731224b).

- [ ] **[OPTIONAL]** Kindly correct the elements in red to be similar to the Figma template 👇🏽 
image

File: style.css [hunk 7]
```diff
@@ -532,3 +533,255 @@ input[type="submit"]:disabled {
...
+
+  #contact {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 15px;
+  }
```
Comment (franclobo) [inline style.css]: - [ ] **[OPTIONAL]** Please 🙏🏽 pay attention to the suggerences from the last code reviewer:

> Kindly make sure to make all the changes from [here](https://www.loom.com/share/76c98f42d74846389b37e6769731224b).

- [ ] **[OPTIONAL]** Kindly correct the elements in red to be similar to the Figma template 👇🏽 
image
- Figma design
image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors ✔️ 
- Correct GitHub flow ✔️ 
- Descriptive PR ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] **[OPTIONAL]** Please 🙏🏽 replace your data in the corresponding section of the README file. (Author 1, we can replace it with our name) 👇🏽 
image

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ngufor-emmanuella** [comment]
Everything in the previous reviews is marked as optional ,however I tried to fix alot in the project thanks

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @Ngufor-emmanuella  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ngufor-emmanuella** [comment]
thank you

**Pull request merged**